modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/GroupResourceManager.java

```diff
@@ -230,9 +230,8 @@ public SCIMResponse delete(String id, UserManager userManager) {
      * @return
      */
     @Override
-    public SCIMResponse listWithGET(UserManager userManager, String filter, int startIndex,
-                                    int count, String sortBy, String sortOrder, String domainName,
-                                    String attributes, String excludeAttributes) {
+    public SCIMResponse listWithGET(UserManager userManager, String filter, int startIndex, int count, String sortBy,
```

Comment:
Why do we need to change the method with the signature including `int count`, since we intend to use the method with the signature including 'Integer count' ?

Reply:
I have not changed the signature. I used WSO2 formatter on this method signature. It has corrected the formatting issues in the signature.

Reply:
I have not changed the signature. I used WSO2 formatter on this method signature. It has corrected the formatting issues in the signature.